docs/omero-release-process.rst

```diff
@@ -14,19 +14,26 @@ The release process uses GitHub actions, make sure that the actions are active b
 Register CVE
 ^^^^^^^^^^^^
 
-To be added
+As soon as a CVE is identified, create a security advisory on `GitHub <https://github.com/ome/openmicroscopy/security/advisories>`_.
+The work to fix the CVE will be done using the private copy of `ome/openmicroscopy <https://github.com/ome/openmicroscopy/>`_ and the private copies of the Java components.
+The release process needs to eb adjusted in that case.
 
 Release process
 ^^^^^^^^^^^^^^^
 
 Source code release
 -------------------
 
-To make a new release:
+To make a new public release:
```

Comment:
What's the difference between a public and a private release? Why would you do one vs another?